dist/backdraft.js

```diff
@@ -1644,9 +1651,15 @@ _.extend(Plugin.factory, {
           var headerGroupDataIndex = columnConfig.headerGroupDataIndex;
           var columnGroupConfig = _.findWhere(columnGroups, { "headerGroupDataIndex" : headerGroupDataIndex } );
 
+          if (columnGroupConfig === undefined || headerGroupDataIndex === undefined) {
```

Comment:
in this case any null/false/undefined should go into this error block right? I'd do:

```
if (!columnGroupConfig || !headerGroupDataIndex) {
```